src/test/resources/smoke-suite.xml

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Smoke suite" thread-count="1">
+    <test name="Chrome group">
+        <classes>
+            <class name="com.github.emilshina.testcases.AuthorizationTests">
+                <methods>
+                    <include name="userShouldBeAuthorizedWithValidCredentials"/>
+                    <include name="userShouldNotBeAuthorizedWithInvalidLogin"/>
+                    <include name="userShouldNotBeAuthorizedWithInvalidPassword"/>
+                </methods>
+            </class>
+            <class name="com.github.emilshina.testcases.MenuTests"/>
+        </classes>
+    </test>
+</suite>
```

Comment:
check new line presence in all java and configuration files to avoid issues with some editors and VSC...

Reply:
Can I use `git config --global core.autocrlf input` command for this? Or how should it be done in a right way?

Reply:
Auto conversion feature is more about mac / linux vs windows users' correct line endings support. It will work when you already have a new line. But in you case lots of files don't have new lines at the end at all. This feature won't make things worse, but you still need to add new lines to your files.